### PR TITLE
label_sync: Add WG Naming label

### PR DESCRIPTION
**Pull request by justaugustus**
Required of WG formation in https://github.com/kubernetes/community/pull/4884.

Signed-off-by: Stephen Augustus <saugustus@vmware.com>

cc: @zacharysarah @celestehorgan

### CODE REVIEW

File: label_sync/labels.yaml [hunk 1]
```diff
@@ -603,6 +603,12 @@ default:
       target: both
       prowPlugin: label
       addedBy: anyone
+    - color: d2b48c
+      description: Categorizes an issue or PR as relevant to WG Naming.
```
Comment (sftim) [inline label_sync/labels.yaml:607]:
```suggestion
      description: Categorizes an issue or PR as relevant to wg-naming.
```
to match the other label descriptions.

Comment (justaugustus) [reply label_sync/labels.yaml:607]: @sftim --

ref: 
- https://kubernetes.io/docs/contribute/style/style-guide/#kubernetes-io-word-list
- https://twitter.com/stephenaugustus/status/1175798240987537408?s=19

Comment (justaugustus) [reply label_sync/labels.yaml:607]: (That said, I can fix up the other references in a follow-up)

Comment (spiffxp) [reply label_sync/labels.yaml:607]: Opened https://github.com/kubernetes/test-infra/issues/18147 for a followup

**Pull request merged**